src/js/sdk.tsx

```diff
@@ -76,8 +76,11 @@ export function init(passedOptions: ReactNativeOptions): void {
     stackParser: stackParserFromStackParserOptions(passedOptions.stackParser || defaultStackParser),
     beforeBreadcrumb: safeFactory(passedOptions.beforeBreadcrumb, { loggerMessage: 'The beforeBreadcrumb threw an error' }),
     initialScope: safeFactory(passedOptions.initialScope, { loggerMessage: 'The initialScope threw an error' }),
-    tracesSampler: safeTracesSampler(passedOptions.tracesSampler),
   };
+  if ('tracesSampler' in options) {
```

Comment:
why is this necessary? also, the impl of `safeTracesSampler` looks strange to my untrained (in TS/JS) eyes, it's basically `if (tracesSampler) { ... } else { return tracesSampler; }`
probably totally fine, just trying to understand & learn :)

Reply:
ok yeah that's what I assumed - and we want to make sure we don't set a falsy value and therefore guard it with the added `if`?

Reply:
Yes, because if we set undefined key is created on the options object which can cause issues with the key existence checks.